borg/archiver.py

```diff
@@ -1076,6 +1107,34 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='REPOSITORY', nargs='?', default='',
                                type=location_validator(archive=False))
 
+        subparser = subparsers.add_parser('key-export', parents=[common_parser],
```

Comment:
@enkore as that would be first subcommand, i do not think it is a good idea for 1.0-maint.
and at the time we introduce subcommands, we could refactor some other stuff also, e.g. debug-* commands.